internal/release/promote/interactive_prompt_provider.go

```diff
@@ -164,18 +168,20 @@ func alignColumns(lines []string) []string {
 	return strings.Split(result, "\n")
 }
 
-func (i *InteractivePromptProvider) ConfirmCreatingPromotionPullRequest() (bool, error) {
-	actions := []string{"Create pull request", "Cancel"}
+func (i *InteractivePromptProvider) SelectCreatingPromotionPullRequest() (answer string, err error) {
+	var selectedAction string
+	actions := []string{Draft, Ready, Cancel}
```

Comment:
let's make Ready the first option! I think it should be the most used, so for least annoyance it should be easiest to reach

Reply:
Changed